exonum-java-binding-time-service/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.exonum.binding</groupId>
+        <artifactId>exonum-java-binding-parent</artifactId>
+        <version>0.4-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>exonum-java-binding-time-service</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Exonum Time Service</name>
+    <url>https://exonum.com/doc/version/latest/advanced/time/</url>
+
+    <properties>
+        <ejb-core.nativeLibPath>${project.parent.basedir}/exonum-java-binding-core/rust/target/debug
+        </ejb-core.nativeLibPath>
+        <junit.jupiter.version>5.3.2</junit.jupiter.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.exonum.binding</groupId>
+                <artifactId>exonum-java-binding-bom</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.exonum.binding</groupId>
+            <artifactId>exonum-java-binding-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.exonum.binding</groupId>
+            <artifactId>exonum-java-testing</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+            </plugin>
+
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <argLine>
+                        ${jacoco.args}
+                        -Djava.library.path=${ejb-core.nativeLibPath}
+                        ${java.vm.assertionFlag}
+                    </argLine>
+                </configuration>
+            </plugin>
+        </plugins>
```

Comment:
Just to confirm: this project won't be released if we do it now, will it?

Reply:
Thanks, added `<skip>true</skip>` for this module.